src/sass/Fabric.scss

```diff
@@ -12,8 +12,10 @@
 @import './Animation';
 @import './BrandIcon';
 @import './Color';
+@import './Font.Definitions';
 @import './Font';
 @import './Grid';
+@import './IconFont';
```

Comment:
Maybe `Icon.Definition` to match the font definitions file? The word 'definition' is already bothering me but `Font.Font` would be far worse.